### ISSUE TITLE
000001.txt  not found

### ISSUE THREAD

**Issue by sareerulamin** [jiayannan/VITT#4]
Hello sir. I could find your traing data annotaions. the images data are there in cfg/ccmcpe.json folder but i got the error" OSError: MOT17/Labels_with_ids/train/MOT17-02-SDP/img1/000001.txt not found"
please guide me regarding this.

**Comment by tdchua** [comment]
Hello! To prepare for this dataset, please take a look at this repository
https://github.com/Zhongdao/Towards-Realtime-MOT/blob/master/DATASET_ZOO.md

ViTT uses the same dataset format as the one in the repository I shared.

**Comment by sareerulamin** [comment]
Could you help me by making the above dataset formate?

**Comment by tdchua** [comment]
Oh, can you be specific on what you do not know or understand from the link i sent you?

**Comment by sareerulamin** [comment]
I want to run this repo as it is. But I couldn't. How to structure mot17 dataset like The link you shared before,

**Comment by sareerulamin** [comment]
I want to run this repo as it is. But I couldn't. How to structure the mot17 dataset like The link you shared before.

**Comment by tdchua** [comment]
You will have to do it yourself like create the folders, then use the tools to put the correct files into them. Which dataset are you having trouble in?

**Comment by sareerulamin** [comment]
Mot17 dataset

**Comment by tdchua** [comment]
> Mot17 dataset

I see. What happened after downloading and unzipping the dataset file?

**Comment by sareerulamin** [comment]
Could you run on mot17 dataset?

**Comment by tdchua** [comment]
> Could you run on mot17 dataset?

Yes, I can. I set the root path on the cfg/ccmpe.json file to point to my directory with the datasets.

**Comment by sareerulamin** [comment]
Can you share with me your dataset folder structure and and path to the dataset?

**Comment by sareerulamin** [comment]
Because i got the open error.

**Comment by tdchua** [comment]
![Screen Shot 2022-03-29 at 10 03 23 PM](https://user-images.githubusercontent.com/23580577/160617423-6d098e67-1a8a-4e7d-8c40-1e8422566b12.png)
Here you go

**Comment by sareerulamin** [comment]
When i downloaded mot17 dataset, its not look like your directory. How you got label with id filder.

**Comment by tdchua** [comment]
> When i downloaded mot17 dataset, its not look like your directory. How you got label with id filder.

Through the use of google my friend, https://github.com/Zhongdao/Towards-Realtime-MOT/issues/108

**Comment by sareerulamin** [comment]
> > Could you run on mot17 dataset?
> 
> Yes, I can. I set the root path on the cfg/ccmpe.json file to point to my directory with the datasets.

Thank you sir, could you please share with me your cfg/ccmpe.jason screen shot?

**Comment by tdchua** [comment]
> > > Could you run on mot17 dataset?
> > 
> > 
> > Yes, I can. I set the root path on the cfg/ccmpe.json file to point to my directory with the datasets.
> 
> Thank you sir, could you please share with me your cfg/ccmpe.jason screen shot?
<img width="535" alt="Screen Shot 2022-03-31 at 8 06 15 PM" src="https://user-images.githubusercontent.com/23580577/161041539-b53d9c7a-6955-4255-a8bc-12bc8f290df9.png">

**Comment by sareerulamin** [comment]
Thank you for your reply, sir, I tried a lot but did not succeed. I would
be very grateful if you share the code, which you used to run the
repository using the mot17 dataset.
I am looking forward to hearing from you a positive response.
Regards
Sareer Ul Amin

**Comment by tdchua** [comment]
> 

I just used the train.py provided with the repository after changing the paths to the datasets.

**Comment by ghost** [comment]
hello sir how are you? 
i got an error while runing the code. 
can you please help me out?
@tdchua

**Comment by tdchua** [comment]
> hello sir how are you? i got an error while runing the code. can you please help me out? @tdchua

Please post the error message here, so I can help
